src/System.Private.CoreLib/src/Internal/Cryptography/Sha1.Unix.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Internal.Cryptography
+{
+    internal static class Sha1
+    {
+        public static byte[] ComputeSha1(byte[] data)
+        {
+            // TODO - CORERT - Add a CoreLibNative_ComputeSha1() entrypoint to System.Private.CoreLib.Native
+            //  and P/Invoke to that. 
+            throw new NotImplementedException();
```

Comment:
@AtsushiKan I do not think it make sense to add dependency from the CoreLib to crypto stack. We should just have a dumb version of dumb local SHA1 implementation to make this work - exactly like it is done in CoreCLR. This is not on any security sensitive path, so dumb local SHA1 implementation is ok. 

If you do not want to re-implement it in CoreRT, feel free to just copy the C++ impl from CoreCLR (https://github.com/dotnet/coreclr/blob/master/src/vm/sha1.cpp) and compile it as part of the native piece.